reporter/src/main/scala/com/typesafe/tools/mima/lib/analyze/Analyzer.scala

```diff
@@ -130,7 +130,7 @@ private[analyze] class TraitAnalyzer extends Analyzer {
   override def analyzeNewClassMethods(oldclazz: ClassInfo, newclazz: ClassInfo): List[Problem] = {
     val res = collection.mutable.ListBuffer.empty[Problem]
 
-    for (newmeth <- newclazz.concreteMethods if !oldclazz.hasStaticImpl(newmeth)) {
+    for (newmeth <- newclazz.concreteMethods if newmeth.isDeferred && !oldclazz.hasStaticImpl(newmeth)) {
```

Comment:
This looks weird, how can `newclazz.concreteMethods` contain a method satisfying `newmeth.isDeferred`?